### PR TITLE
fix display bundle on token page, add bundle information on "market" …

### PR DESCRIPTION
**Pull request by punkrussian72**
…and "my tokens" page

### CODE REVIEW

File: src/components/OfferCard/OfferCard.tsx [hunk 1]
```diff
@@ -56,6 +60,20 @@ export const OfferCard: FC<TTokensCard> = ({ offer, testid }) => {
     tokenId: offer?.tokenId.toString()
   }), [currentChainId, offer]);
 
+  const tokensCount = useMemo(() => {
```
Comment (SergeyOkishev) [inline src/components/OfferCard/OfferCard.tsx:63]: 1. move to utils
2. check existing count calculation on tokenPage (no need to duplicate this code)

File: src/components/TokensCard/TokensCard.tsx [hunk 2]
```diff
@@ -130,14 +153,26 @@ export const TokensCard: FC<TTokensCard> = ({ collectionId, tokenId, offer, test
             {collectionTitle}
           </Text>
         </a>
+        {isBundleState && <BundleItemsCount>
```
Comment (SergeyOkishev) [inline src/components/TokensCard/TokensCard.tsx]: copypaste, extract component instead

File: src/components/TokensCard/TokensCard.tsx [hunk 3]
```diff
@@ -79,6 +77,31 @@ export const TokensCard: FC<TTokensCard> = ({ collectionId, tokenId, offer, test
     return {};
   }, [collectionId, tokenId, collection, token, api]);
 
+  useEffect(() => {
```
Comment (SergeyOkishev) [inline src/components/TokensCard/TokensCard.tsx:80]: utils

seems overcomplicated for effect

**Review by SergeyOkishev — CHANGES_REQUESTED** [review]

File: src/components/OfferCard/OfferCard.tsx (hunk 1 above)
Comment (punkrussian72) [reply src/components/OfferCard/OfferCard.tsx:63]: fixed

File: src/components/TokensCard/TokensCard.tsx (hunk 2 above)
Comment (punkrussian72) [reply src/components/TokensCard/TokensCard.tsx]: fixed

File: src/components/TokensCard/TokensCard.tsx (hunk 3 above)
Comment (punkrussian72) [reply src/components/TokensCard/TokensCard.tsx:80]: fixed

**Review by SergeyOkishev — APPROVED** [review]

**Pull request merged**